sdks/python/tests/unit/evaluation/test_evaluate.py

```diff
@@ -64,13 +67,16 @@ def say_task(dataset_item: dataset_item.DatasetItem):
             "construct_online_streamer",
             mock_construct_online_streamer,
         ):
-            evaluation.evaluate(
-                dataset=mock_dataset,
-                task=say_task,
-                experiment_name="the-experiment-name",
-                scoring_metrics=[metrics.Equals()],
-                task_threads=1,
-            )
+            with mock.patch.object(
```

Comment:
Just to point out, the test does not verify whether a notification was actually printed.

Reply:
I didn't know how to add tests for it but I manually verified it works, we can add a test in the future if it fails